src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -684,7 +684,7 @@ xclLoadAxlf(const axlf *buffer)
   std::string dtbo_path("");
 
 #ifndef __HWEM__
-  auto is_pr_platform = (buffer->m_header.m_mode == XCLBIN_PR ) ? true : false;
+  auto is_pr_platform = (buffer->m_header.m_mode == XCLBIN_PR || buffer->m_header.m_actionMask & AM_LOAD_PDI) ? true : false;
```

Comment:
Kind of strange to do `if (true) true; else false`;  :-)  Maybe drop the ternary?

Reply:
yeah.. Fixed it Soren. Thank you.